src/frontend/apps/e2e/__tests__/app-desk/common.ts

```diff
@@ -46,3 +46,45 @@ export const createTeam = async (
 
   return randomTeams;
 };
+
+export const addNewMember = async (
+  page: Page,
+  index: number,
+  role: 'Admin' | 'Owner' | 'Member',
+  fillText: string = 'test',
+) => {
+  const responsePromiseSearchUser = page.waitForResponse(
+    (response) =>
+      response.url().includes(`/users/?q=${fillText}`) &&
+      response.status() === 200,
+  );
+  await page.getByLabel('Add members to the team').click();
+  const inputSearch = page.getByLabel(/Find a member to add to the team/);
+
+  // Select a new user
+  await inputSearch.fill(fillText);
+
+  // Intercept response
+  const responseSearchUser = await responsePromiseSearchUser;
+  const users = (await responseSearchUser.json()).results as {
+    name: string;
+  }[];
+
+  console.log('index' + index, users[index].name);
```

Comment:
to be removed

Reply:
[fixup! 🛂(app-desk) add TeamActions component](https://github.com/numerique-gouv/people/pull/148/commits/da4456fb3ae86356220a8c96357d6c650d004a77)